statbot/sql.py

```diff
@@ -584,6 +665,60 @@ def typing(self, trans, channel, user, when):
         trans.execute(ins)
         self.typing_cache[key] = True
 
+    # Status
+    def status_change(self, trans, member):
+        timestamp = datetime.now()
+        key = (timestamp, member.id)
+
+        if self.status_cache.get(key, None):
+            self.logger.debug("Status change lookup is up-to-date")
+            return
+
+        self.logger.info(f"Inserting status change event for user {member.id}")
+        ins = self.tb_statuses \
+                .insert() \
+                .values({
+                    'timestamp': timestamp,
+                    'user_id': member.id,
+                    'user_status': UserStatus.convert(member.status),
+                })
```

Comment:
i think it may be preferable to just wrap this whole thing into parentheses. it's probably a preference thing, but I think it's more readable than `\` `\`  `\` everywhere. up to you

Reply:
That style is used everywhere in the codebase, it would take a lot of changes to adjust it.